modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java

```diff
@@ -120,6 +120,10 @@ protected void reindex(String className, long[] companyIds)
 				ReindexBackgroundTaskConstants.SINGLE_START, companyId,
 				companyIds);
 
+			if (_log.isInfoEnabled()) {
+				_log.info("Reindexing started [companyId=" + companyId + "]");
```

Comment:
@wcao20170619 lets add className to the messages as well, thanks!

Reply:
@BryanEngler , I am not sure what you want(example?) currently, we have the following.
[ReindexPortalBackgroundTaskExecutor:57] Reindexing started [companyId=20097]
2022-09-19 22:09:53.530 INFO [liferay/background_task-2][ReindexPortalBackgroundTaskExecutor:76] Reindexing ended [companyId=20097]

Reply:
@wcao20170619 My understanding is that we want the name of the entity class being reindexed to be also logged, say `com.liferay.journal.model.JournalArticle`

Reply:
i think for now only add the class name in the 2 logging statements in ReindexSingleIndexerBackgroundTaskExecutor.java

Reply:
example: [companyId=20097,className=com.liferay.journal.model.JournalArticle]

Reply:
OK, ONLY in ReindexSingleIndexerBackgroundTaskExecutor.java.

Reply:
Do we need to change the following too for the spellchecker?
Start indexing dictionary for com/liferay/portal/search/dependencies/spellchecker/en_US.txt [companyId=0]